addons/mrp/tests/test_order.py

```diff
@@ -4916,6 +4916,50 @@ def test_mo_without_resource_calendar(self):
         self.assertEqual(mo.state, 'done')
         self.assertEqual(mo.workorder_ids[0].duration_expected, 1440.0)
 
+    def test_plan_depending_workorder(self):
+        """
+        Checks that modifying the real duration of a workorder depending on other workorders that have already been started works.
+        """
+        # create a production with two workorders, one depending on the other
+        self.env.user.groups_id += self.env.ref('mrp.group_mrp_routings')
+        self.env.user.groups_id -= self.env.ref('mrp.group_mrp_workorder_dependencies')
+
+        self.product_depends = self.env['product.product'].create({
+            'name': 'depends',
+        })
+
+        self.bom_depends = self.env['mrp.bom'].create({
+            'product_id': self.product_6.id,
+            'product_tmpl_id': self.product_6.product_tmpl_id.id,
+            'product_uom_id': self.uom_dozen.id,
+            'ready_to_produce': 'asap',
+            'consumption': 'flexible',
+            'product_qty': 1.0,
+            'allow_operation_dependencies': True,
+            'operation_ids': [
+                (0, 0, {'name': 'Cutting Machine', 'workcenter_id': self.workcenter_1.id, 'time_cycle': 12, 'sequence': 1}),
+                (0, 0, {'name': 'Weld Machine', 'workcenter_id': self.workcenter_1.id, 'time_cycle': 18, 'sequence': 2}),
+            ],
+            'type': 'normal',
+            'bom_line_ids': []
+        })
+        self.bom_depends.operation_ids[1].blocked_by_operation_ids = [(4, self.bom_depends.operation_ids[0].id)]
+
+        # Create a production, then start the first workorder and try to modify the duration of the second one
+        production_form = Form(self.env['mrp.production'])
+        production_form.bom_id = self.bom_depends
+        production = production_form.save()
+        production.action_confirm()
+        production.workorder_ids[0].button_start()
+
+        production_form = Form(production)
+        with production_form.workorder_ids.edit(1) as wo:
+            wo.duration = 15 # in 15 minutes
```

Comment:
```suggestion
            wo.duration = 15
```
I think that's obvious enough :smile: